packages/inference/test/InferenceClient.spec.ts

```diff
@@ -2408,18 +2408,18 @@ describe.skip("InferenceClient", () => {
 			const client = new InferenceClient(env.HF_CLARIFAI_KEY ?? "dummy");
 
 			HARDCODED_MODEL_INFERENCE_MAPPING["clarifai"] = {
-				"Qwen/Qwen3-235B-A22B-Instruct-2507": {
+				"clarifai/DeepSeek-V3_1": {
 					provider: "clarifai",
-					hfModelId: "Qwen/Qwen3-235B-A22B-Instruct-2507",
-					providerId: "Qwen/Qwen3-235B-A22B-Instruct-2507",
+					hfModelId: "clarifai/DeepSeek-V3_1",
+					providerId: "deepseek-ai/deepseek-chat/models/DeepSeek-V3_1",
```

Comment:
The ids must match names of models on the HuggingFace Hub (see https://huggingface.co/deepseek-ai/DeepSeek-V3.1)

```suggestion
				"deepseek-ai/DeepSeek-V3.1": {
provider: "clarifai",
					hfModelId: "deepseek-ai/DeepSeek-V3.1",
					providerId: "deepseek-ai/deepseek-chat/models/DeepSeek-V3_1",
```